src/pytest/test_config.py

```diff
@@ -0,0 +1,8 @@
+def testConfig(env):
+    env.cmd('ft.create', 'idx', 'SCHEMA', 'test', 'TEXT', 'SORTABLE')
+    env.expect('ft.config', 'help', 'idx').equal([])
+    env.expect('ft.config', 'set', 'MINPREFIX', 1).equal('OK')    
+    env.expect('ft.config', 'set', 'MINPREFIX', 1, 2).equal('EXCESSARGS')    
+    env.expect('ft.config', 'set', 'MINPREFIX', 'str').equal('Success (not an error)')  
```

Comment:
This can not be correct

Reply:
I wondered about it but it fails at RSConfig_SetOption() and call QueryError_GetError()

Should I change to RedisModule_ReplyWithError()?

Reply:
@MeirShpilraien 
There is no real mechanism that checks for correct parsing.
Some SET functions return RM_OK or RM_ERR but some return char*.
A bit messy!

Reply:
`Success (not an error)` is a bug. This shouldn't appear to the user.

Reply:
https://github.com/RediSearch/RediSearch/issues/956

Reply:
@ashtul lets remove this check as it is wrong check and we have a ticket to fix it.

Reply:
commented tests out with TODO label